node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs');
+const randomNum = Math.random().toString() + '\n';
```

Comment:
Since you are concating the number to a string you don't need to use `toString` it will be converted to a string already.

Reply:
ok gotcha